src/index.js

```diff
@@ -0,0 +1,132 @@
+const state = {
+  temp: 68
+};
+
+let isFahrenheit = true;
+
+const vibe = {
+  sheet: 2
+}
+
+let isReset = false;
+
+const changeTemp = (direction) => {
+  if (direction === 'up') {
+    state.temp += 1;
+  } else if (direction === 'down') {
+    state.temp -= 1;
+  } else if (direction === null) {
+    pass
+  }
+
+  const temperature = document.querySelector("#temperature");
+  if (isFahrenheit) {
+    temperature.textContent = `${state.temp}℉`;
+  } else {
+    temperature.textContent = `${(state.temp -32) * 5/9}℃`;
+  }
+  changeTempFontColor();
+}
+
+
+const swapStyleSheet = (event) => {
+
+  if (vibe.sheet === 1 || vibe.sheet === 2) {
+      vibe.sheet += 1;
+  } else if (vibe.sheet === 3) {
+      vibe.sheet = 1;
+  }
+
+  document.getElementById("vibe").setAttribute("href", `styles/vibe${vibe.sheet}.css`)
+}
+
+const changeTempFontColor = () => {
+  if (state.temp <= 49) {
+    color = "#3F50B5";
+    landscape = "🤧⛄️🌲⛄️🛷🌲⛷🗻🌲⛸🏂⛄️🍂❄️";
+  } else if (state.temp >= 50 && state.temp<= 59) {
+    color = "#00BCD4";
+    landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else if (state.temp >= 60 && state.temp  <= 69) {
+    color = "#FEC106";
+    landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if (state.temp >= 70 && state.temp  <= 79) {
+    color = "#FF9802";
+    landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else {
+    color = "#F44336";
+    landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  }
+  document.getElementById("temperature").style.color = color;
+  document.getElementById("landscapeContainer").textContent = landscape;
+}
+
+const submitInput = () => {
+  
+  const subheaderText = document.querySelector("#inputFormContents");
+  const inputButton = document.querySelector("#inputButton");
+  
+  if (isReset) {
```

Comment:
As a user, I would rather there be a separate button to reset the values. As it is, it's a little deceptive that the input fields are still there, but the button becomes a reset button. So If I want to change the values, I have to perform a reset in between. But if I didn't notice that the button turned into a reset button, and had typed my next input, it will get cleared!